src/components/Layout/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC, ReactNode } from 'react';
+
+import Head from 'next/head';
+import Navbar from '@/components/Navbar';
+import Footer from '@/components/Footer';
+
+type LayoutProps = {
+    title: string;
+    children: ReactNode;
+};
+
+const Layout: FC<LayoutProps> = ({ title, children }) => {
```

Comment:
h-container class on the <main> element is not defined, so the browser will ignore it. This means that the height of the <main> element will be determined by the content inside of it, which could cause it to overlap the footer on small screens.

Reply:
https://github.com/Real-Dev-Squad/tiny-site-frontend/pull/18#discussion_r1345773798